### PR TITLE
Add metabase chart

### PR DESCRIPTION
**Pull request by pmint93**
The helm chart for Metabase

### CODE REVIEW

File: stable/metabase/templates/secrets.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+type: Opaque
+data:
+  username: {{ .Values.database.username | b64enc | quote }}
+  password: {{ .Values.database.password | b64enc | quote }}
+{{- end }}
+---
```
Comment (unguiculus) [inline stable/metabase/templates/secrets.yaml]: One file per resource, please.

**Review by unguiculus — CHANGES_REQUESTED** [review]
Thanks for the PR. 

The `app` labels is  inconsistent across resources. Please have a look at our best practices, especially regarding standard labels.

https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/labels.md

**Comment by pmint93** [comment]
@unguiculus I've just made a few changes as your request, sorry for my mistake 😄

**Review by unguiculus — APPROVED** [review]

**Comment by unguiculus** [comment]
Good work, thanks. Don' be sorry. :wink:

**Pull request merged**